src/Radio/RadioGroup.js

```diff
@@ -47,6 +47,26 @@ export default class RadioGroup extends Component {
     radioGroup: PropTypes.object,
   };
 
+  static changeFocus(currentFocusIndex = 0, event, radios) {
```

Comment:
Again, WDYT? Module function or static method? Always feels strange to me using static methods that are only used by instances.